codebuild/bin/s2n_codebuild.sh

```diff
@@ -65,6 +65,23 @@ if [[ -n "$S2N_NO_PQ" ]]; then
     CMAKE_PQ_OPTION="S2N_NO_PQ=True"
 fi
 
+source codebuild/bin/s2n_apache2.sh
+APACHE_CERT_DIR="$(pwd)/tests/pems"
+
+run_integration_v2_tests() {
+    apache2_start "${APACHE_CERT_DIR}"
```

Comment:
Don't we usually run one integrationv2 test at a time in the CI? That would mean we'd spin apache up every time, even though only 1 out of like 18 tests needs it. Is there a way we can start apache only if the integration test will require it?

Reply:
That's true. Starting apache is fast, but it probably makes sense to check to see if it's needed first. I added a check that looks at the `TOX_TEST_NAME` variable, and enables it if the renegotiate test is being run.